Zend/zend_builtin_functions.c

```diff
@@ -1557,27 +1559,68 @@ static void debug_backtrace_get_args(zend_execute_data *call, zval *arg_array) /
 					 */
 					while (i < first_extra_arg) {
 						zend_string *arg_name = call->func->op_array.vars[i];
-						zval *arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
-						if (arg) {
-							ZVAL_DEREF(arg);
-							Z_TRY_ADDREF_P(arg);
-							ZEND_HASH_FILL_SET(arg);
+						zval arg;
+						zval *old_arg = zend_hash_find_ex_ind(call->symbol_table, arg_name, 1);
+						zend_attribute *attribute = zend_get_parameter_attribute_str(
+							call->func->op_array.attributes,
+							"sensitiveparameter",
+							sizeof("sensitiveparameter") - 1,
+							i
+						);
+
+						last_was_sensitive = attribute != NULL;
```

Comment:
Nit: This should be called `is_sensitive`, as it refers to the current and not the last parameter.

Reply:
Thanks. The variable was originally created to properly handle variadics, but has since lost its meaning during the refactoring.

While researching this I also figured out how to cleanly detect variadic parameters (`fn_flags & ZEND_ACC_VARIADIC` instead of that `<=` check` + comment) and also fixed the logic to access `.common.attributes` instead of `.op_array.attributes` which I believe should be a little cleaner as we also need to handle internal functions.